### PR TITLE
Update in.m3u

### PR DESCRIPTION
**Pull request by vishveshjain**
Added all sony network channels

### CODE REVIEW

**Review by Alstruit — CHANGES_REQUESTED** [review]
[CONTRIBUTING.md#stream-description-scheme](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#stream-description-scheme)

**Review by vishveshjain — COMMENTED** [review]
Changes made as per [iptv](https://github.com/iptv-org/iptv/tree/master)
/CONTRIBUTING.md

File: streams/in.m3u [hunk 1]
```diff
@@ -516,6 +516,50 @@ http://103.81.104.118/hls/stream5.m3u8
...
+https://dai.google.com/linear/hls/event/V9h-iyOxRiGp41ppQScDSQ/master.m3u8
+#EXTINF:-1 tvg-id="SonySportsTen3.in",Sony Sports Ten 3 HD (1080p)
+https://dai.google.com/linear/hls/event/ltsCG7TBSCSDmyq0rQtvSA/master.m3u8
+#EXTINF:-1 tvg-id="SonySportsTen4.in",Sony Sports Ten 4 HD (1080p)
+https://dai.google.com/linear/hls/event/tNzcW2ZhTVaViggo5ocI-A/master.m3u8
+#EXTINF:-1 tvg-id="SonySportsTen5.in",Sony Sports Ten 5 HD (1080p)
+https://dai.google.com/linear/hls/event/Sle_TR8rQIuZHWzshEXYjQ/master.m3u8
+#EXTINF:-1 tvg-id="SonyKALHindi.us",Sony Kal (1080p)
+https://spt-sonykal-1-us.lg.wurl.tv/playlist.m3u8
+#EXTINF:-1 tvg-id="SonyEntertainmentTelevisionAsiaHD.in",	Sony Entertainment Television Asia HD (1080p)
```
Comment (Alstruit) [inline streams/in.m3u:529]:
```suggestion
#EXTINF:-1 tvg-id="SonyEntertainmentTelevisionAsiaHD.in",Sony Entertainment Television Asia HD (1080p)
```

File: streams/in.m3u [hunk 2]
```diff
@@ -516,6 +516,50 @@ http://103.81.104.118/hls/stream5.m3u8
...
+https://dai.google.com/linear/hls/event/dBdwOiGaQvy0TA1zOsjV6w/master.m3u8
+#EXTINF:-1 tvg-id="	SonySAB.in",Sony SAB (1080p)
```
Comment (Alstruit) [inline streams/in.m3u:531]:
```suggestion
#EXTINF:-1 tvg-id="SonySAB.in",Sony SAB (1080p)
```

**Review by Alstruit — APPROVED** [review]

**Review by Alstruit — APPROVED** [review]

**Comment by vishveshjain** [comment]
Hi still waiting for 1 more approval.

**Review by BellezaEmporium — APPROVED** [review]
LGTM.

**Pull request merged**